File: archive/(9)0 bytes XD/intp.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question('Input code > ', (answer) => {
    if (code === '0') {
        console.log('0');
        return;
    } 
    if (code == '1') {
        while (true) {
            console.log('1');
        }
    }
    rl.close();
});

File: (9)0 bytes XD/intp.rs
use std::io; 

fn main() {
    println!("Code > "); 
    let mut user_input = String::new();
    io::stdin()
        .read_line(&mut user_input)
        .expect("Failed to read line"); 
    let input = user_input.trim();
    if input == "0" {
        println!("0");
    }
    if input == "1" {
        while 7 == 7 {
            println!("1");
        }
    }
}